Ex#1/ecasa_inv/src/main/java/com/avangenio/ecasa_inv/domain/ProductType.java

```diff
@@ -0,0 +1,52 @@
+package com.avangenio.ecasa_inv.domain;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+
+@Entity
+public class ProductType {

```

Comment:
We are using the entities as DTOs here. Good as a shortcut but not the best practice. Just worth noticing it.